packages/eyes.sdk/src/EyesBase.js

```diff
@@ -904,6 +916,12 @@
         return this._runningSession;
     };
 
+    EyesBase.prototype.getAndSaveRenderingInfo = function () {
```

Comment:
This isn't used anywhere (and also using a function that's not defined). Did you leave it here by mistake?